frontend/src/components/custom-tools/prompt-card/PromptCardItems.jsx

```diff
@@ -263,7 +263,6 @@ function PromptCardItems({
             isCoverageLoading={isCoverageLoading}
             isEditingTitle={isEditingTitle}
             setIsEditingTitle={setIsEditingTitle}
-            enableEdit={enableEdit}
```

Comment:
@jagadeeswaran-zipstack Are we removing the `enableEdit` part completely ?

Reply:
Yes, enableEdit used only for edit button on prompt card which is removed now